test/scenarios-periodics/el92-src@cncf-conformance.sh

```diff
@@ -41,8 +41,7 @@ run_sonobuoy() {
     ~/go/bin/sonobuoy run \
         --mode=certified-conformance \
         --dns-namespace=openshift-dns \
-        --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default \
-        --e2e-parallel=y 
+        --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default
```

Comment:
If we're merging this PR, please exclude the CNCF job from pre-submits as discussed on the Slack channel.

Reply:
/hold